src/components/activitylog.js

```diff
@@ -2,7 +2,7 @@ import escapeHtml from 'escape-html';
 import Events from '../utils/events.ts';
 import globalize from '../lib/globalize';
 import { ServerConnections } from 'lib/jellyfin-apiclient';
-import dom from '../scripts/dom';
+import dom from '../utils/dom.js';
```

Comment:
_nit_ all the other places we don't have the `.js` on the import.

Reply:
I really don't know why VS code occasionally adds this when files are moved :man_facepalming: